chart.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
I think this was a test file as well that you were toying with? Probably don't need it in the repo at this point?